page03_contact.php

```diff
@@ -1,105 +1,123 @@
+		<style>
+			h1,h2 {
+				font-family: 'Alegreya';
+				font-size: 35w;
+				color: white;
+
+			}
+		</style>
+
 
 	</head>
 	<body>
 
 		<!-- Navbar (sit on top) -->
 		<div class="w3-top">
-			<div class="w3-bar w3-grey w3-padding w3-card">
+			<div class="w3-bar w3-white w3-padding w3-card">
 				<img src="images/placeholder.jpg" width="40" height="40">
 				<a href="indexV1.php" class="w3-bar-item w3-button"><b>Soundwave</b> </a>
 
-
 				<!-- Float links to the right -->
 				<div class="w3-right">
 					<a href="indexV1.php" class="w3-bar-item w3-button w3-xxlarge fa fa-home"></a>
 					<a href="playlistV1.php" class="w3-bar-item w3-button w3-xxlarge material-icons">playlist_add</a>	
 					<a href="contact.php" class="w3-bar-item w3-button w3-xxlarge material-icons" >contact_support</a>
+					<?php
+						if ($_SESSION["login_user"] == "Graeme"){
+						echo "<a href='03_add_user(settings).php' class='w3-bar-item w3-button w3-xxlarge material-icons'>settings</a>";}
+						?>
 					<a href="signout.php" class="w3-bar-item w3-button w3-xxlarge material-icons">logout</a>
 				</div>
 			</div>
 		</div>
 
 
-		<div class="w3-black">
-			<div class="w3-container w3-content w3-padding-64" style="max-width:100%">
+		<!-- background container; colour orange -->
+		<div class="container2" style="max-width:100%">
 
 
-				<div class="w3-content w3-container w3-padding-32 w3-display-middle-large w3-black w3-image">
+			<div class="w3-content w3-container w3-padding-32 w3-display-middle-large w3-image">
 
-					<?php
-					//Get the connection to the database in phpMyAdmin
-					require_once("communityproject_mysqli.php");
-					?>
-
-					<!-- Form -->
-					<form action="connect.php" method="post" class="w3-container w3-card-4 w3-black w3-text-orange w3-margin"> 
-						<h2 class="w3-center">CONTACT US</h2>
-
-						<div class="w3-row w3-section">
-							<div class="w3-col" style="width:50px"><i class="w3-xxlarge fa fa-user"></i></div>
-							<div class="w3-rest">
-								<input class="w3-input w3-border" id="name" name="name" type="text" placeholder= "Name">
-							</div>
-						</div>		
-
-						<div class="w3-row w3-section">
-							<div class="w3-col" style="width:50px"><i class="w3-xxlarge fa fa-envelope-o"></i></div>
-							<div class="w3-rest">
-								<input class="w3-input w3-border" id="email" name="email" type="text" placeholder="Email">
-							</div>
-						</div>
+				<?php
+//Get the connection to the database in phpMyAdmin
+require_once("communityproject_mysqli.php");
+?>
 
-						<div class="w3-row w3-section">
-							<div class="w3-col" style="width:50px"><i class="w3-xxlarge fa fa-phone"></i></div>
-							<div class="w3-rest">
-								<input class="w3-input w3-border" id="phone" name="phone" placeholder="A contact number" >
-							</div>
-						</div>
+				<!-- Form -->
+				<form action="connect.php" method="post" class="w3-container w3-text-white w3-margin"> 
+					<h1 class="w3-center">CONTACT US</h1>
```

Comment:
change heading2 to heading1:
h1
font-family: 'Alegreya';
font-size: 35w;
color: white;